src/components/social-share/social-share.scss

```diff
@@ -5,4 +5,7 @@
   & > * {
     margin: 4px;
```

Comment:
Adding `display: flex` here will do the trick.

Reply:
Also, we could change that margin to `margin: 7px 4px;` instead to get closer to the design

Reply:
I have implemented the requested changes.